src/router/routes/cube/blog.ts

```diff
@@ -212,7 +213,7 @@ export const getBlogPostsForCubeHandler = async (req: Request, res: Response) =>
           `Cube Cobra Blog: ${cube.name}`,
           cube.description,
           cube.image.uri,
-          `https://cubecobra.com/cube/blog/${encodeURIComponent(req.params.id)}`,
+          `${baseUrl}/cube/blog/${encodeURIComponent(req.params.id)}`,
```

Comment:
I thought about removing the base URL here and appending within generateMeta but decided not to because it seemed like the meta URL doesn't have to always be to cubecobra.com. The image URI meta is only a couple times (eg sample pack) pointing to cube cobra